inclusive-design/wecount.inclusivedesign.ca
# feat: homepage update

## Description

1. Change the landing page video and upload the blue image provided by @uttaraghodke. All a rectangular blue bar on top as in the Figma design.
2. Replace the image used in the yellow card in the landing page.
3. Add SVG filter for images in different UIO themes.

## Steps to test

1. Go to the landing page
2. See the Youtube player is replaced with an image
3. See different filters are applied to the image in different themes of UIO
